modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/ImageMosaicConfigHandler.java

```diff
@@ -1346,6 +1348,27 @@ private void createPropertiesFiles(MosaicConfigurationBean mosaicConfiguration)
             properties.setProperty(Prop.SUGGESTED_FORMAT, cachedFormat.getClass().getName());
         }
 
+        if (catalogConfigurationBean.isCog()) {
+            properties.setProperty(Prop.IS_COG, Boolean.toString(catalogConfigurationBean.isCog()));
+            CogConfigurationBean cogBean = catalogConfigurationBean.getCogBean();
+            if (cogBean != null) {
+                String rangeReader = cogBean.getRangeReader();
+                if (rangeReader != null) {
+                    properties.setProperty(Prop.COG_RANGE_READER, rangeReader);
```

Comment:
All these new properties should be documented somewhere (I get this is just a draft, just saying that doc updates should be included in the final version of the PR).

Reply:
@aaime I was actually planning to add them in the geoserver doc as part of the incoming GeoServer PR since most part of these fields and values is defined there.
https://docs.geoserver.org/stable/en/user/data/raster/imagemosaic/configuration.html#configuration-files

Will it make sense?

Reply:
Hum... I thought there was more documentation about mosaics in GeoTools, but what I can find points to the GeoServer docs: https://docs.geotools.org/stable/userguide/library/coverage/mosaic.html
Looks good to me, makes little sense to keep the same documentation duplicated in two places.